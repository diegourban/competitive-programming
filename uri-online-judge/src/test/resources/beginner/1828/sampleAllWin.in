10
pedra tesoura
pedra lagarto
papel pedra
papel Spock
tesoura papel
tesoura lagarto
lagarto papel
lagarto Spock
Spock pedra
Spock tesoura