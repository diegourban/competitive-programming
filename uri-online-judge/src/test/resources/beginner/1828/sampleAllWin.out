Caso #1: Bazinga!
Caso #2: Bazinga!
Caso #3: Bazinga!
Caso #4: Bazinga!
Caso #5: Bazinga!
Caso #6: Bazinga!
Caso #7: Bazinga!
Caso #8: Bazinga!
Caso #9: Bazinga!
Caso #10: Bazinga!
